common/transport/amqp/src/amqp_message.ts

```diff
@@ -88,6 +89,12 @@ export class AmqpMessage {
       amqpMessage.applicationProperties['iothub-ack'] = message.ack;
     }
 
+    if (message.diagnosticPropertyData) {
+      // Codes_SRS_NODE_IOTHUB_AMQPMSG_26_001: [If message.diagnosticPropertyData is truthy, the function shall add diagnostic information as AMQP message annotation.]
```

Comment:
>Codes_SRS_NODE_IOTHUB_AMQPMSG_26_001 [](start = 9, length = 36)

Please add the corresponding requirement in common/transport/amqp/devdoc/amqp_message.md

Reply:
added